packages/devextreme/js/__internal/ui/context_menu/m_context_menu.ts

```diff
@@ -1130,6 +1133,10 @@ class ContextMenu extends MenuBase {
   hide(): Promise<unknown> {
     return this.toggle(false);
   }
+
+  _postProcessRenderItems() {
+    this._dimensionChanged(false);
```

Comment:
Looks not very logical that we call `dimensionChanged` after all items are rendered. Cannot we split the code another way? Like extract necessary code from `_dimensionChanged` to a new method and reuse it here

Reply:
changed logic - now we do not change height after render, instead height calculated and submenu visiblility changed yo visible after render is done